package.json

```diff
@@ -25,7 +25,8 @@
     "@angular/cdk": "^5.2.5",
     "@angular/common": "^5.0.0",
     "@angular/core": "^5.0.0",
-    "@angular/forms": "^5.0.0"
+    "@angular/forms": "^5.0.0",
+    "ngx-bootxtrap": "2.0.4"
```

Comment:
Might be worth prepending this with `^`?

Reply:
Done.